### DISCUSSION TITLE
Mastodon

### DISCUSSION THREAD

**Discussion by voidrender** [transitive-bullshit/nextjs-notion-starter-kit#399] [Ideas]
It would be neat to have support for Mastodon in the social footer using the [link verification](https://docs.joinmastodon.org/user/profile/) `rel` anchor so it can be shown as verified on your Mastodon profile.

```
<a rel="me" href={$MASTODON_PROFILE_LINK}>Mastodon</a>
```

Seems relatively straightforward:
- New configuration variable (optional) for Mastodon profile link
- Conditionally rendered Mastodon icon in social footer with above var as href

**Comment by JIeJaitt** [comment]
Hi, Excuse me, do you know how to achieve this function? No matter what I try I can't get this to work！
<img width="874" alt="image" src="https://user-images.githubusercontent.com/77219045/208232078-047b507b-0b30-463d-95d5-876f0316dfdd.png">
